test/plugin/test_out_google_cloud.rb

```diff
@@ -503,10 +499,19 @@ def expected_operation_message2
     OPERATION_MESSAGE2
   end
 
-  # Both expected and actual are Ruby hashes that represent JSON
-  # objects.
-  # This method has a different implementation at the gRPC side.
-  def assert_hash_equal_json(expected, actual)
-    assert_equal expected, actual, "expected: #{expected}\nactual: #{actual}"
+  def enable_grpc
+    false
+  end
+
+  def default_nano_value
+    0
+  end
+
+  def timestamp_seconds(timestamp)
+    timestamp['seconds']
+  end
+
+  def entry_timestamp_nanos(timestamp)
```

Comment:
Same as above, I think it'd be clearer to rename this to `timestamp_nanos`. Is that possible?

Reply:
Sure. Renamed.